exit.go

```diff
@@ -549,6 +549,25 @@ func (te *TunaExit) StartReverse(shouldReconnect bool) error {
 			continue
 		}
 
+		reverseIP := tcpConn.RemoteAddr().(*net.TCPAddr).IP
+		reverseTCP := reverseMetadata.ServiceTcp
+		if len(reverseTCP) > 0 {
+			go func() {
+				net.DialTimeout(tcp, fmt.Sprintf("%s:%d", reverseIP.String(), reverseTCP[0]), defaultReverseTestTimeout)
+			}()
+
+			session.SetDeadline(time.Now().Add(defaultReverseTestTimeout))
+
+			_, err = session.AcceptStream()
+			if err != nil {
+				log.Println("Couldn't accept stream test conn from reverse entry:", err)
+				time.Sleep(1 * time.Second)
+				continue
+			}
+
+			session.SetDeadline(time.Time{})
```

Comment:
Is this required to set it back?

Reply:
Yes, otherwise `session.Accept()` will always return error after 3s